### ISSUE TITLE
Patching __init__.py fails when module docstring is in double quotes

### ISSUE THREAD

**Issue by genotrance** [adang1345/delvewheel#42]
Two issues I found while working on my new project - first is that delvewheel requires you to use triple quotes for the module docstring. Python does not require this and my module has a single line in double quotes. I don't see why this is needed but delvewheel complains if this is the case. But before I even discovered this, a secondary issue threw me off.

Many of my function docstrings use triple quotes since they have multi-line descriptions. Considering my module docstring was in double quotes, delvewheel placed the patch template after the first multi-line function docstring, midway in the file, breaking the verification after patching.

Basically - delvewheel should not require module docstrings to be in triple quotes, and should not get distracted with function docstrings which are triple quoted.
